### ISSUE TITLE
clarify that accesskey makes element focusable but doesn't change its role

### ISSUE THREAD

**Issue by chaals** [w3c/html#296]
from [bugzilla](https://www.w3.org/Bugs/Public/show_bug.cgi?id=23616)

**Comment by chaals** [comment]
Except that [doesn't work in browsers](https://chaals.github.io/accesskey/tests/accesskey-focus-manual.html) ;(

**Comment by chaals** [comment]
Moving to incubation for now.
